src/main/interface/settings.c

```diff
@@ -609,7 +609,7 @@ const clivalue_t valueTable[] = {
     { "rc_smoothing_derivative_type",VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_RC_SMOOTHING_DERIVATIVE_TYPE }, PG_RX_CONFIG, offsetof(rxConfig_t, rc_smoothing_derivative_type) },
 #endif // USE_RC_SMOOTHING_FILTER
 
-    { "fpv_mix_degrees",            VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 50 }, PG_RX_CONFIG, offsetof(rxConfig_t, fpvCamAngleDegrees) },
+    { "fpv_mix_degrees",            VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 90 }, PG_RX_CONFIG, offsetof(rxConfig_t, fpvCamAngleDegrees) },
```

Comment:
I am wondering if we should constrain this to 89 degrees? With a setting of 90 degrees, roll and yaw will be effectively swapped, not sure if that makes any sense?

Reply:
Interesting idea. Personally it does make any difference. While it is somewhat equivalent to remapping rx input, i still think that a one degree difference is not something that will make a significant difference.

Reply:
Yeah, admittedly it won't make a big difference. What I am wary about is that this might cause problems like division by 0 somewhere down the signal path, so limiting it to 89 degrees will be the safe bet.